src/applications/static-pages/static-pages-entry.js

```diff
@@ -1,93 +1,79 @@
-import 'platform/polyfills';
+// Node modules.
 import LazyLoad from 'vanilla-lazyload/dist/lazyload';
 import * as Sentry from '@sentry/browser';
-
-import createCommonStore from 'platform/startup/store';
-import startSitewideComponents from 'platform/site-wide';
-import { VA_FORM_IDS } from 'platform/forms/constants';
-
+// Relative imports.
 import './analytics';
+import './sass/static-pages.scss';
```

Comment:
This is all just reorganizing to make it more legible 🙂 (except the ask-va part)